compiler/damlc/lib/DA/Cli/Damlc/Packaging.hs

```diff
@@ -277,9 +292,9 @@ generateAndInstallInstancesPkg thisSdkVer templInstSrc opts dbPathAbs projectPac
             opts' <-
                 mkOptions $
                 opts
-                    { optWriteInterface = True
+                    { optWriteInterface = False
+                    , optIfaceDir = Nothing
```

Comment:
Just to be sure, this is because
> -- [...] This is _only_
    -- relevant for `compile` which at the moment is only used for building daml-stdlib
    -- and daml-prim.

?

Reply:
yep, I’ll remove it completely from the `Options` record in a follow up PR. the current state makes no sense.